gen/templates/model.go

```diff
@@ -40,9 +40,11 @@ type {{camelCase .Name}} struct {
 {{- end}}
 {{- range .ChildClasses}}
 {{- if eq .Type "single"}}
+{{- if len .Attributes -}}
```

Comment:
Why do we need the length check? Shouldn't it otherwise iterate over an empty list with the same result?